server/src/modules/notifications/agents/agent.ts

```diff
@@ -0,0 +1,24 @@
+import { Notification } from '../entities/notification.entities';
+import {
+  NotificationAgentConfig,
+  NotificationAgentKey,
+  NotificationType,
+} from '../notifications-interfaces';
+
+export interface NotificationPayload {
```

Comment:
It would be nice to include some metadata in this payload where relevant, such as the collection name, and/or the media item name. The webhook only has access to the subject and message which isn't super useful.